test/compression.js

```diff
@@ -301,6 +311,34 @@ describe('compression()', function () {
       .expect(200, done)
   })
 
+  describeHttp2('http2', function () {
+    it('should work with http2 server', function (done) {
+      var server = createHttp2Server({ threshold: 0 }, function (req, res) {
+        res.setHeader('Content-Type', 'text/plain')
+        res.end('hello, world')
+      })
+      server.on('listening', function () {
+        var client = createHttp2Client(server.address().port)
+        var request = client.request({
+          'Accept-Encoding': 'gzip'
+        })
+        request.on('response', function (headers) {
+          assert.equal(headers['content-encoding'], 'gzip')
+        })
+        request.on('error', function (error) {
+          console.error('An error event occurred on a http2 client request.', error)
```

Comment:
No one is going to comb through the CI logs after every run or upgrade... do these need to be here vs just letting an error crash the process? If they are preventing an expected error, it shouldn't be logging to the screen, as there should be no extra output when everything was actually OK.